jasper_server/ir_actions.py

```diff
@@ -1,113 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
It seems it's used for all the reports
https://github.com/odoo/odoo/blob/10.0/odoo/addons/base/ir/ir_actions.py#L87